support/src/traits.rs

```diff
@@ -68,3 +68,10 @@ pub trait VersionMigratorTrait<T>: Sized {
 pub trait GenerateBenchmarkOrigin<OuterOrigin, AccountId, SubjectId> {
 	fn generate_origin(sender: AccountId, subject: SubjectId) -> OuterOrigin;
 }
+
+/// Trait that allows types to implement a worst case value for a type,
+/// only when running benchmarks.
+#[cfg(feature = "runtime-benchmarks")]
+pub trait GetWorstCase {
```

Comment:
What is this about?

Reply:
It's a trait that we can use to generate worst case for a given parameter, when the parameter does not depend on a benchmark step. For instance in this PR, I had to generate the worst case for an asset DID, and because the mock runtime and the actual runtime define a different type for the asset DID, the benchmark would not be able to generate an instance that would satisfy both. So I introduced a new trait that is implemented both by the test type and by the actual type.
Albi did something similar also for signature verifications, as in https://github.com/KILTprotocol/kilt-node/blob/develop/support/src/signature.rs#L38=.